docs/configuration.md

```diff
@@ -246,6 +248,26 @@ blockrelay:
       # With a factor of 0 bids from this builder will be ignored.
       factor: 0
 
+# builderclient provides timeout configuration for builder/relay communication.
+# Vouch communicates with MEV-boost relays for validator registration, bid fetching, and block unblinding.
+# Different operations have different performance characteristics and may benefit from different timeout values.
+builderclient:
+  # Global timeout for all builderclient operations. Falls back to the global 'timeout' value if not set.
+  timeout: '10s'
```

Comment:
Is the default value really 2s if this isn't set? If so we should change the value to that.